noggin/tests/unit/conftest.py

```diff
@@ -87,7 +87,6 @@ def ipa_testing_config(vcr_session):
         except python_freeipa.exceptions.BadRequest as e:
             if not e.message == "no modifications to be performed":
                 raise
-            raise
```

Comment:
Oh yeah I think that's my fault, I've put this in as a debug line and forgot to take it off